test/hoctests/tests/test_kschan.py

```diff
@@ -1,35 +1,227 @@
 from neuron import h, gui
+from neuron.expect_hocerr import expect_err
```

Comment:
I think I did briefly try that before, but some of the tests only pass because of fixes that are in this PR -- so one would have to hoist both the test **and** some fixes, which is probably possible but may not be trivial.